python/bolt-mcp/src/bolt_mcp/oauth/config.py

```
@@ -154,3 +154,23 @@ def redirect_uri_allowed(self, registered: list[str] | None, redirect_uri: str)
         """Whether ``redirect_uri`` is permitted. Default: exact match against registered
         values (no open redirect). Override only with great care."""
         return redirect_uri in (registered or [])
+
+    def code_redirect_response(self, redirect_url: str) -> Any:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `code_redirect_response`

(ANN401)